generator/csv/fields.csv

```diff
@@ -936,7 +936,7 @@ SaveEventExecState,keyinput_left,f,Boolean,0x24,False,0,0,
 SaveEventExecState,keyinput_right,f,Boolean,0x25,False,0,0,
 SaveEventExecState,keyinput_up,f,Boolean,0x26,False,0,0,
 SaveEventExecState,keyinput_timed,f,Boolean,0x29,False,0,0,
-SaveEventExecState,unused_wait_for_key_or_enter,f,Boolean,0x2a,False,0,0,If enabled; an event waits for either the confirmation key to be pressed or one of the keys defined by KeyInputProc before continuing. This flag seems to be unused though since it is never written to (keyinput_wait is used instead).
```

Comment:
Whats up with this comment? I tested with rm2k3e and it definately uses this chunk when you do a "Wait for Key Input" command. That is the wait command variant that waits for input, not a key input proc.